bigbluebutton-html5/imports/ui/components/whiteboard/component.jsx

```diff
@@ -518,6 +518,13 @@ const Whiteboard = React.memo(function Whiteboard(props) {
         delete remoteShape.questionType;
         toAdd.push(remoteShape);
       } else if (!isEqual(localShape, remoteShape) && prevShape) {
+        if (
+          (remoteShape?.meta?.createdBy === currentUser?.userId && remoteShape?.meta?.updatedBy === currentUser?.userId)

```

Comment:
Could we do something like

`const remoteShapeMeta = remoteShape?.meta;`
and then

`if (remoteShapeMeta && ....`

just to avoid 8 checks if remoteShape, if meta, etc